### PR TITLE
Fixes #16906: Be able to save in LDAP a Policy Server Target

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/16906

### CODE REVIEW

**Comment by ElaadF** [comment]
PR rebased

**Comment by ElaadF** [comment]
Commit modified

**Comment by ElaadF** [comment]
Commit modified

**Comment by ElaadF** [comment]
Commit modified

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/repository/ldap/LDAPNodeRepository.scala [hunk 1]
```diff
@@ -63,6 +64,7 @@ class WoLDAPNodeRepository(
   , mapper              : LDAPEntityMapper
   , ldap                : LDAPConnectionProvider[RwLDAPConnection]
   , actionLogger        : EventLogRepository
+  , rudderDit           : RudderDit
```
Comment (VinceMacBuche) [inline webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/repository/ldap/LDAPNodeRepository.scala]: Arf you don't need rudderDit

**Pull request merged**